data/sql/updates/pending_db_world/rev_1611180684314340400.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611180684314340400');
+
+DELETE FROM `spell_script_names` WHERE `spell_id` = 75731;
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
```

Comment:
in that case, you could have used "replace into" as it affect only one row, although it's only working in mysql based databases if i recall right

(not required to edit it, was just for info)

Reply:
yeye, I you're right.
But personally I, and I know other members also, prefers DEL/INS